### ISSUE TITLE
PowerShell script - with space in path / new process

### ISSUE THREAD

**Issue by MichelZ** [webprofusion/certify#662]
Hi

We are using Certify `6.0.12.0` and are trying to execute a PowerShell Script deployment task.
The Task fails with the error:

```
2023-12-11 07:20:37.264 +01:00 [INF] ---- Performing Task [On-Demand or Manual Execution] :: Deploy to RDP Gateway (Custom) ----
2023-12-11 07:20:37.265 +01:00 [INF] Task [Deploy to RDP Gateway (Custom)] :: Task is being has been forced to run. Normal status would be [Task is enabled but will not run because primary request unsuccessful.]
2023-12-11 07:20:37.265 +01:00 [INF] Executing command via PowerShell
2023-12-11 07:20:37.634 +01:00 [ERR] Error: D:\Program : The term 'D:\Program' is not recognized as the name of a cmdlet, function, script file, or operable 
Error: program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
Error: At line:1 char:1
Error: + D:\Program Files\CertifyTheWeb\Scripts\Internal\Script-Wrapper.ps1 -s ...
Error: + ~~~~~~~~~~
Error:     + CategoryInfo          : ObjectNotFound: (D:\Program:String) [], CommandNotFoundException
Error:     + FullyQualifiedErrorId : CommandNotFoundException
Warning: Script exited with the following ExitCode: 1
```

As you can see, CertifyTheWeb is installed in `D:\Program Files`
First I thought it was my script that was stored there as well, but i've moved it to a path without spaces and it still complains about the same error.

The Task configuration is:
```
Task Type: Run PowerShell Script
Authentication: Local (as current service user)
Program/Script:   <path to script>
Pass Result as First Arg: Yes
Impersonation LogonType: Interactive
Arguments (options): Empty
Script Timeout Mins: Empty
Launch New Process: Yes
```

Thanks & Cheers
Michel

**Comment by webprofusion-chrisc** [comment]
Thank @MichelZ we had a fix for this in v6.0.11 (it's specific to the wrapper script used for launching in a new process). Please note that you definitely must not put your own scripts anywhere under the  \Program Files\CertifyTheWeb location because the installer will (should) delete them on the next update.

Can you try uninstalling the app and re-install? It looks to me like perhaps not all the apps files have copied properly when you last updated.

**Comment by MichelZ** [comment]
@webprofusion-chrisc 
Thanks. I have reinstalled it, all files seem to be from 2023-11-25 (e.g. Script-Wrapper.ps1, all in /Plugins/, all Certify.*.exe/dll in the main directory). Server was also rebooted.
Still the same issue. 

The target script is not in the CertifyTheWeb directory, but somewhere else in `D:\Program Files\<ourcompany>\CertifyTheWebScripts`

I have even tried deleting the Deployment Task and recreating it, without success.

**Comment by webprofusion-chrisc** [comment]
Thanks @MichelZ we'll investigate. If you are a licensed customer please submit a support ticket to support {at} certifytheweb.com to track this properly, we don't generally use GitHub issues for customer support.

**Comment by webprofusion-chrisc** [comment]
Closing as fixed in v6.0.13
